lib/Dialect/FIRRTL/Import/FIRAnnotations.cpp

```diff
@@ -315,6 +315,14 @@ bool circt::firrtl::fromJSON(json::Value &value,
       return false;
     StringRef targetStrRef = optTarget.getValue();
 
+    auto circuitFieldEnd = targetStrRef.find_first_of('|');
```

Comment:
This could be a straight `CircuitTarget` annotation with a target like `~Fooo`. Can you update this to handle the case of it _not_ ending in `|`?

Reply:
Thanks for catching this! Fixed in 13a169e